Gruntfile.js

```diff
@@ -27,13 +27,25 @@ module.exports = function (grunt) {
             }
         },
         jsonlint: {
-            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json"]
+            all: ["package.json", ".jshintrc", "src/js/*.json", "tests/**/*.json", "demos/**/*.json"]
```

Comment:
Are there json files in "src/js" directory? What's the reason to change "src/**/*.json" to "src/js/*.json"?